packages/security/oidc-provider-plugin-wildcard-redirect-uri/readme.md

```
@@ -20,7 +20,7 @@
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
   <a href="https://tsed.io/getting-started/">Getting started</a>
   <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
-  <a href="https://api.tsed.io/rest/slack/tsedio/tsed">Slack</a>
+  <a href="https://slack.tsed.io">Slack</a>
```

Markdownlint (0.35.0)

Element: a
Inline HTML

(MD033, no-inline-html)